testfiles/CMakeLists.txt

```diff
@@ -323,6 +323,11 @@ ADD_SIMULATION_TEST(IDF_FILE House-2FurnaceAC-UniformPLR.idf EPW_FILE USA_IL_Chi
 ADD_SIMULATION_TEST(IDF_FILE HybridVentilationControl.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 ADD_SIMULATION_TEST(IDF_FILE HybridVentilationControlGlobalSimple.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 ADD_SIMULATION_TEST(IDF_FILE HybridZoneModel.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE HybridModel_4Zone_Solve_Infiltration_with_CO2.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE HybridModel_4Zone_Solve_Infiltration_with_Temperature.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE HybridModel_4Zone_Solve_PeopleCount_with_HumidityRatio.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE HybridModel_4Zone_Solve_PeopleCount_with_Temperature_no_HVAC.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE HybridModel_4Zone_Solve_PeopleCount_with_Temperature_with_HVAC.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
```

Comment:
5 new example files - that's a lot. @Myoldmopar do we want to move any of these to start with underscore (not included in distribution)? Or perhaps combine these such that each zone illustrates a different method?